File: app/src/main/java/com/example/tetrisapp/util/DateTimeUtil.java
package com.example.tetrisapp.util;

import java.text.DateFormat;
import java.text.SimpleDateFormat;
import java.util.Date;
import java.util.TimeZone;

public class DateTimeUtil {

    public static String toISOString(Date date) {
        TimeZone tz = TimeZone.getTimeZone("UTC");
        DateFormat df = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
        df.setTimeZone(tz);
        return df.format(date);
    }
}

File: app/src/main/java/com/example/tetrisapp/util/DateTimeUtil.kt
package com.example.tetrisapp.util

import java.text.DateFormat
import java.text.SimpleDateFormat
import java.util.*

object DateTimeUtil {
    fun toISOString(date: Date): String {
        val tz = TimeZone.getTimeZone("UTC")
        val df: DateFormat = SimpleDateFormat(
            "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'",
            Locale.getDefault(Locale.Category.FORMAT)
        )
        df.timeZone = tz
        return df.format(date)
    }
}